app/src/main/java/com/example/musicroulette/MainActivity.java

```diff
@@ -50,10 +51,26 @@ public void RequestToken() {
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
-        getMenuInflater().inflate(R.menu.bottom_navigation_menu, menu);
```

Comment:
This seems to be the biggest issue. This commit seems fine as long as you know this is correct.